bridges/snowbridge/primitives/core/src/location.rs

```diff
@@ -23,23 +23,30 @@ pub type AgentId = H256;
 
 /// Creates an AgentId from a Location. An AgentId is a unique mapping to an Agent contract on
 /// Ethereum which acts as the sovereign account for the Location.
-/// Resolves Polkadot locations (as seen by Ethereum) to unique `AgentId` identifiers.
-pub type AgentIdOf = HashedDescription<
-	AgentId,
-	(
-		DescribeHere,
-		DescribeFamily<DescribeAllTerminal>,
-		DescribeGlobalPrefix<(DescribeTerminus, DescribeFamily<DescribeTokenTerminal>)>,
-	),
->;
+#[deprecated(note = "Use LocationHashOf instead.")]
+pub type AgentIdOf =
+	HashedDescription<AgentId, (DescribeHere, DescribeFamily<DescribeAllTerminal>)>;
 
 pub type TokenId = H256;
 
 /// Convert a token location (relative to Ethereum) to a stable ID that can be used on the Ethereum
 /// side
+#[deprecated(note = "Use LocationHashOf instead.")]
 pub type TokenIdOf = HashedDescription<
 	TokenId,
-	DescribeGlobalPrefix<(DescribeTerminus, DescribeFamily<DescribeTokenTerminal>)>,
+	DescribeForEthereum<(DescribeTerminus, DescribeFamily<DescribeSnowbridgeTerminal>)>,
+>;
+
+/// Resolves Polkadot locations (as seen by Ethereum) to unique `H256` identifiers.
+/// V1 converter is left here for backwards compatibility
```

Comment:
As discussed in today's meet, we need to preserve compatibility for several locations that were encoded in the previous V1 format.

Are there tests that cover:
* The agent id for /Para(AssetHub) remains in the V1 format.
* That the token id for /GlobalConsensus(Polkadot)/Here remains in the V1 format?